roles/redhat_subscription/tasks/main.yaml

```diff
@@ -21,6 +22,27 @@
 # the lookup()
 # via http://stackoverflow.com/a/34239020
 #
+  - name: Get the current commit to find the correct origin file to manipulate later...
+    command: rpm-ostree status --json
+    register: rpm_output
+
+  - name: Convert deployment information into jinja2 json
+    set_fact:
+      json: "{{ rpm_output.stdout|from_json }}"
+
+  - name: Set version and commit if deployment 0 is booted
+    set_fact:
+      booted_commit: "{{ json['deployments'][0]['checksum'] }}"
+    when: json['deployments'][0]['booted']
+
+  - name: Set version and commit if deployment 1 is booted
+    set_fact:
+      booted_commit: "{{ json['deployments'][0]['checksum'] }}"
+    when: json['deployments'][0]['booted']
```

Comment:
This looks like a copy/paste error.  I think you want to use the deployment at index 1.